build.gradle

```diff
@@ -452,6 +452,24 @@ task yarn(dependsOn: mapIntermediaryJar, type: JavaExec) {
 	jvmArgs "-Xmx2048m"
 }
 
+task yarnUnpicked(dependsOn: "unpickIntermediaryJar", type: JavaExec) {
+	logger.lifecycle(":yarnUnpicked is deprecated, unpick is enabled by default")
```

Comment:
This isnt right, wont this print even if you dont call it?

Id just remove it tbh, or at the very least depend on `yarn` and print the message.

Reply:
```groovy
task yarnUnpicked(dependsOn: "yarn") {
   doLast {
      logger.lifecycle(":yarnUnpicked is deprecated, unpick is enabled by default")
   }
}
```

Is what you want I think, im not even sure if it should be noted as deprecated?